COMMUNITY-ROLES.md

```diff
@@ -89,108 +89,129 @@ using one of the template messages below as a base.
 
 ### For adding new collaborators
```

Comment:
While we're at it, I'll suggest changing these headings removing the leading "For". "Adding new collaborators" sounds way better as a heading IMHO.

```suggestion
 ### Adding new collaborators
```

Reply:
Sounds good, will change.